src/fides/api/ctl/routes/generate.py

```diff
@@ -69,6 +73,7 @@ def target_matches_type(cls, values: Dict) -> Dict:
         """
         target_type = (values.get("target"), values.get("type"))
         valid_target_types = [
+            ("aws", "datasets"),
```

Comment:
I'm thinking this might be better off as `dynamodb` instead of `aws` to reduce some of the confusion and keep in line with `bigquery` (it doesn't say `gcp` for instance)

Reply:
I agree, I'd rather have it be in line with the name of the connector.

Reply:
Cool, will clean this up 👍🏽 comment further down has me in a bit of a pickle with the inconsistency between cli and api but they are different patterns to maybe it makes sense 🤷🏽 more opinions needed I think in the future

Reply:
We should also update the example in the docstring `('aws', 'datasets')` is a valid example now and that is too close to the `('aws', 'datasets')` in the invalid example.

Reply:
Updated all in 5bc94373add7218f9f2aeac5afd2ceb7ab85351d